opensuse-tomcat-jre21-image/pom.xml

```diff
@@ -170,4 +165,35 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <!-- profile for docker proxy settings -->
+        <profile>
+            <id>docker-proxy</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Maybe this can be made even smarter by enabling the profile only if the `HTTP_PROXY` environment variable is present, but we can look into that for another commit.